csfieldguide/templates/interactives/lzw-compression.html

```diff
@@ -4,31 +4,22 @@
 {% load static %}
 
 {% block html %}
-<div id="interactive-lzw-compression" class="row mt-3">
-  <div class="col-12 text-center">
-    <h1 id="heading">{% trans "LZW Compression" %}</h1>
-  </div>
-  <div id="content" class="container">
-
-		<div id="interactive-lzw-compression">
-
+<div id="container">
+    <div id="interactive-lzw-compression">
+        <h3 id="heading">{% trans "LZW Compression" %}</h3>
+        <div id="interactive-lzw-compression-wrapper">
             <div id="interactive-lzw-compression-left">
                 <div class="interactive-lzw-compression-to-compress-block" id="interactive-lzw-compression-to-compress">
                     <p>{% trans "Message to compress" %}:</p>
-                    <textarea id="interactive-lzw-compression-input">
-Pease porridge hot, pease porridge cold,
-Pease porridge in the pot, nine days old;
-Some like it hot, some like it cold,
-Some like it in the pot, nine days old.</textarea>
-                    <button id="interactive-lzw-compression-compress-button" type="button">
+                    <textarea class="form-control" rows="5" id="interactive-lzw-compression-input"></textarea>
```

Comment:
If we delete the placeholder message it pops back up on the next line until a key is pressed.
![image](https://user-images.githubusercontent.com/16066822/56932020-38fcac00-6b36-11e9-8519-3ba70cbdeb31.png)

Reply:
Is this to prevent users from trying to compress an empty string?

Reply:
that's amusing how it adds a line, have fixed. Also that was the original purpose but in hindsight the behaviour is slightly confusing so have removed it as placeholder text.